bin/template.jinja

```diff
@@ -13,7 +13,9 @@ parameters:
     type: string
   {%- for param in parameters %}
   {{ param['name'] }}:
+    {%- if param['name'] != "token" %}
```

Comment:
Not a huge fan of having this override here, previously when i had to override for a single method i implemented this here: https://github.com/StackStorm-Exchange/stackstorm-slack/blob/master/bin/generate_openapi.py#L19

Would be probably good to add a "global" override or something here: https://github.com/StackStorm-Exchange/stackstorm-slack/blob/master/bin/generate_openapi.py#L156

Reply:
Ok, I will take a look at that, thanks!

Reply:
Added a simple loop and a constant, removed the previous change to `./bin/template.jinja` (still need to update changelog)
Regenerated actions and now `token['required']` is being overridden to `false`
https://github.com/StackStorm-Exchange/stackstorm-slack/blob/f90ca25c549cd62f7e63939f095abda6a0fac0fb/bin/generate_openapi.py#L164

Does this approach it more how you were seeing it?